src/main.cpp

```diff
@@ -260,7 +286,7 @@ int main(int argc, char *argv[])
     signal(SIGABRT, signal_handler);
 #endif
 
-    LOGI("Starting wayfire");
+    LOGI("Starting wayfire", WAYFIRE_VERSION);
```

Comment:
You should add a space after 'wayfire', at least, like you did [here](https://github.com/WayfireWM/wayfire/pull/490/files#diff-7ec3c68a81efff79b6ca22ac1f1eabbaR66). I would suggest something like `LOGI("Starting wayfire version ", WAYFIRE_VERSION);`